server/embed/config.go

```diff
@@ -348,6 +348,9 @@ type Config struct {
 	// that exist at the same time.
 	// Can only be used if ExperimentalEnableDistributedTracing is true.
 	ExperimentalDistributedTracingServiceInstanceID string `json:"experimental-distributed-tracing-instance-id"`
+	// ExperimentalDistributedTracingSamplingRatePerMillion is the number of samples to collect per million spans.
+	// Defaults to 0.
```

Comment:
I don't think `0` is sensible default. Please provide a value that makes sense when someone enabled tracing for the first time. 

I also think `0` should be forbidden as this is an invalid value, we should not use it to disable tracing as we have a separate flag for it `"experimental-enable-distributed-tracing"`. Configuration flags should be perpendicular (2 flags should not be responsible for enabling/disabling the feature), as so we should not allow setting `experimental-distributed-tracing-sampling-rate` to zero to disable tracing, as users should use `experimental-enable-distributed-tracing` for that.